lib/livebook/notebook/explore/intro_to_vega_lite.livemd

```diff
@@ -28,6 +28,68 @@ alias VegaLite, as: Vl
 
 <!-- livebook:{"branch_parent_index":0} -->
 
+## The Chart cell
+
+Before we get into exploring all the various chart types, let's have
+a look at an awesome feature that comes with `kino_vega_lite` - the
+**Chart** cell!
+
+Coding up a chart usually involves a couple steps. If you don't know
+the API of a particular library, it may be a bit challenging. On the
+other hand, if you know the API in-and-out, it's a rather repetitive
+task. That's where the **Chart** cell comes in, it is a special type
+of cell that magically writes all of the chart code for us. Let's
+give it a try!
+
+First, we need some data to work with, here's a small excerpt from
+the popular "Iris" dataset:
+
+```elixir
+iris = [
+  %{"petal_length" => 5.1, "petal_width" => 1.9, "species" => "Iris-virginica"},
+  %{"petal_length" => 4.0, "petal_width" => 1.3, "species" => "Iris-versicolor"},
+  %{"petal_length" => 1.6, "petal_width" => 0.2, "species" => "Iris-setosa"},
+  %{"petal_length" => 1.6, "petal_width" => 0.2, "species" => "Iris-setosa"},
+  %{"petal_length" => 4.6, "petal_width" => 1.4, "species" => "Iris-versicolor"},
+  %{"petal_length" => 4.8, "petal_width" => 1.8, "species" => "Iris-virginica"},
+  %{"petal_length" => 5.6, "petal_width" => 2.2, "species" => "Iris-virginica"},
+  %{"petal_length" => 5.1, "petal_width" => 1.6, "species" => "Iris-versicolor"},
+  %{"petal_length" => 1.5, "petal_width" => 0.3, "species" => "Iris-setosa"},
+  %{"petal_length" => 4.5, "petal_width" => 1.6, "species" => "Iris-versicolor"}
+]
+
+:ok
+```
+
+Now, to insert a new Chart cell, place your cursor between cells,
+click on the <kbd>+ Smart</kbd> button and select Chart.
+
+You can see an example of that below. Click on the "Evaluate"
+button to see the chart, then see how it changes as you customize
+the parameters.
+
+<!-- livebook:{"attrs":{"chart_title":"Iris","height":200,"layers":[{"chart_type":"point","color_field":"species","color_field_aggregate":null,"color_field_type":null,"data_variable":"iris","x_field":"petal_length","x_field_aggregate":null,"x_field_type":"quantitative","y_field":"petal_width","y_field_aggregate":null,"y_field_type":"quantitative"}],"vl_alias":"Elixir.Vl","width":400},"kind":"Elixir.KinoVegaLite.ChartCell","livebook_object":"smart_cell"} -->
+
+```elixir
+Vl.new(width: 400, height: 200, title: "Iris")
+|> Vl.data_from_values(iris, only: ["petal_length", "petal_width", "species"])
+|> Vl.mark(:point)
+|> Vl.encode_field(:x, "petal_length", type: :quantitative)
+|> Vl.encode_field(:y, "petal_width", type: :quantitative)
+|> Vl.encode_field(:color, "species")
+```
+
+Under cell actions there is a "Source" button, click on it to
+see the source code of your chart. You can even convert it to
+a regular Code cell for further adjustments!
+
+The Chart cell we just saw is in fact one of many **Smart** cells
+that help us automate our workflows ⚡ Not only that, we can create
+custom cells like that, which we explore in detail in the
```

Comment:
```suggestion
The Chart smart cell is one of many **Smart** cells available
in Livebook ⚡ Not only that, you can create your own smart
cells too, which we discuss in the
```

Reply:
In the smart cells guide we consistently use the capitalization "Smart cell", because it's similar to Code and Markdown cells, do we want to downcase there too?

Reply:
Good call. Let's go with "Smart cell" then. :)

Reply:
FTR I'm going with capitalize first, so "Chart smart cell" and "Smart cell" :)